Fortigate fix (#35939)
demisto/content: Packs/FortiGate/ReleaseNotes/2_0_3.md

#### Integrations

##### FortiGate

> - Fixed an issue with **fortigate-list-firewall-policies** command which caused all policies to be displayed when filtering by a policyName that did not have any matches.

Reviewer: ```suggestion
- Fixed an issue where the ***fortigate-list-firewall-policies*** command retrieved all policies when filtering by a policy name that did not match any existing policies.
```
